server/neptune/gateway/event/push_event_handler.go

```diff
@@ -31,9 +36,17 @@ func (p *PushHandler) Handle(ctx context.Context, event Push) error {
 	}
 
 	if !shouldAllocate {
-		// drop the event for now
 		return nil
 	}
 
+	p.Scheduler.Schedule(ctx, func(ctx context.Context) error {
```

Comment:
If i understand this correctly, this `handle` method will signal our deployment workflow with the new revision right?

Reply:
well, yeah it needs to clone the repo, build the root configs, and then signal yeah.